File: app/src/main/java/com/nckh2016/vuduytung/nckh2016/Data/ObjectMonHoc.java
package com.nckh2016.vuduytung.nckh2016.Data;

/**
 * Created by Tung on 24/2/2016.
 */
public class ObjectMonHoc implements Items{
    String mamh;
    String mabm;
    String tenmh;
    Integer tinchi;
    String dieukien;
    String noidung;
    String tailieu;
    String tuchon;
    boolean isHocKy = false;
    double diem = -1;
    boolean bangdiem = false;

    @Override
    public boolean isHocKy() {
        return isHocKy;
    }

    public ObjectMonHoc() {
    }

    public ObjectMonHoc(String mamh, String mabm, String tenmh, Integer tinchi, String dieukien, String noidung, String tailieu, String tuchon){
        this.mamh = mamh;
        this.mabm = mabm;
        this.tenmh = tenmh;
        this.tinchi = tinchi;
        this.dieukien = dieukien;
        this.noidung = noidung;
        this.tailieu = tailieu;
        this.tuchon = tuchon;
    }

    public ObjectMonHoc(String mamh, String mabm, String tenmh, Integer tinchi, String dieukien, String noidung, String tailieu, String tuchon, double diem, boolean bangdiem){
        this.mamh = mamh;
        this.mabm = mabm;
        this.tenmh = tenmh;
        this.tinchi = tinchi;
        this.dieukien = dieukien;
        this.noidung = noidung;
        this.tailieu = tailieu;
        this.tuchon = tuchon;
        this.diem = diem;
        this.bangdiem = bangdiem;
    }

    public ObjectMonHoc(String mamh, double diem, Integer tinchi) {
        this.mamh = mamh;
        this.diem = diem;
        this.tinchi = tinchi;
    }

    public String getMamh() {
        return mamh;
    }

    public void setMamh(String mamh) {
        this.mamh = mamh;
    }

    public String getMabm() {
        return mabm;
    }

    public void setMabm(String mabm) {
        this.mabm = mabm;
    }

    public String getTenmh() {
        return tenmh;
    }

    public void setTenmh(String tenmh) {
        this.tenmh = tenmh;
    }

    public Integer getTinchi() {
        return tinchi;
    }

    public void setTinchi(Integer tinchi) {
        this.tinchi = tinchi;
    }

    public String getDieukien() {
        return dieukien;
    }

    public void setDieukien(String dieukien) {
        this.dieukien = dieukien;
    }

    public String getNoidung() {
        return noidung;
    }

    public void setNoidung(String noidung) {
        this.noidung = noidung;
    }

    public String getTailieu() {
        return tailieu;
    }

    public void setTailieu(String tailieu) {
        this.tailieu = tailieu;
    }

    public String getTuchon() {
        return tuchon;
    }

    public void setTuchon(String tuchon) {
        this.tuchon = tuchon;
    }

    public void setIsHocKy(boolean isHocKy) {
        this.isHocKy = isHocKy;
    }

    public double getDiem() {
        return diem;
    }

    public void setDiem(double diem) {
        this.diem = diem;
    }

    public boolean isBangdiem() {
        return bangdiem;
    }

    public void setBangdiem(boolean bangdiem) {
        this.bangdiem = bangdiem;
    }
}

File: WinForm_CapNhatCTDT/WinForm_CapNhatCTDT/LogicLayer/ObjectMonHoc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinForm_CapNhatCTDT.LogicLayer
{
    class ObjectMonHoc
    {
        private String mamh;
        private String mabm;
        private String tenmh;
        private Int64 tinchi;
        private String dieukien;
        private String noidung;
        private String tailieu;

        public String MaMonHoc { get { return mamh; } set { mamh = value; } }
        public String MaBoMon { get { return mabm; } set { mabm = value; } }
        public String TenMonHoc { get { return tenmh; } set { tenmh = value; } }
        public Int64 TinChi { get { return tinchi; } set { tinchi = value; } }
        public String DieuKien { get { return dieukien; } set { dieukien = value; } }
        public String NoiDung { get { return noidung; } set { noidung = value; } }
        public String TaiLieu { get { return tailieu; } set { tailieu = value; } }

        public ObjectMonHoc()
        {

        }

        public ObjectMonHoc(String mamh, String mabm, String tenmh, Int64 tinchi, String dieukien, String noidung, String tailieu)
        {
            this.mamh = mamh;
            this.mabm = mabm;
            this.tenmh = tenmh;
            this.tinchi = tinchi;
            this.dieukien = dieukien;
            this.noidung = noidung;
            this.tailieu = tailieu;
        }
    }
}
